### PR TITLE
Removal of `config/temperature` for various Xiaomi devices

### PR DESCRIPTION
**Pull request by mattreim**

### CODE REVIEW

File: devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json [hunk 1]
```diff
@@ -46,6 +46,19 @@
         },
         {
           "name": "attr/swversion",
+          "parse": {
```
Comment (ebaauw) [inline devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json:49]: Isn't this already reported through the Xiaomi special attribute?

File: devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json [hunk 2]
```diff
@@ -138,6 +140,19 @@
...
```
Comment (ebaauw) [inline devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json:143]: Isn't this already reported through the Xiaomi special attribute?

**Review by ebaauw — COMMENTED** [review]
I like this; `config/temperature` has been nothing but confusion.  I think we should remove it from all Xiaomi devices and deprecate the resource item (I don't think it's used by other devices).

File: devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json (hunk 1 above)
Comment (mattreim) [reply devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json:49]: Thanks for the tip, I'll try it again.

Comment (mattreim) [reply devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json:49]: It seems to work.

**Comment by SwoopX** [comment]
I definitively support this one as well. However, there's probably more devices out there where this should be done and then the universe will start yelling at us that we must bring this back. As I recall, none of the DDFs initially had that temperature crazyness included at the very beginning and not even Xiaomi is neither offering nor making any use of that value.

**Comment by mattreim** [comment]
Over time it turned out that the sensor didn't recognize the version after all.

**Comment by manup** [comment]
image

Quick note for myself for Xiaomi motion sensor (lumi.sensor_motion.aq2).
Xiaomi special report directly after pairing, before the DDF is matched.

**Comment by jorgenbosman** [comment]
Hi @mattreim, what was the purpose for this PR? There is no linked issue? I recently updated my deconz and now the temperature of my vibration sensor is no longer reported to home assistant. I just stumbled on this PR, so it seems that this is the cause of my issue...

**Comment by mattreim** [comment]
Hello @jorgenbosman, the temperature displayed is the device temperature and does not show the real values. This entity is only used by older Xiaomi devices and only causes confusion. It will no longer be supported by [deCONZ](https://github.com/dresden-elektronik/deconz-rest-plugin-v2/issues/19) in the long term. You should just remove it in Home Assistant.

**Comment by openschwall** [comment]
Actually, I would like to have the temperature from my MCCGQ11LM as it is (better: was) for free. I cannot downgrade to deconz 2.29. again as this is not available in my repo. Is their a way to still retrieve the temperature, for example by a DDF file or another means? Any hint appreciated.

**Comment by openschwall** [comment]
I answer to my own question: you can restore the old behavior by replacing the new .json device files by the old ones which still supply temperature information. Then, in Debian linux, you can hold this state by "sudo apt-mark hold deconz" which avoids updating.

**Pull request merged**